docs/docs/commands/clients.mdx

```diff
@@ -51,16 +48,7 @@ Using config file: /Users/maceo/.sedge.yaml
  3   -                   prysm               teku
```

Comment:
Remove `prysm`

Reply:
for the consensus is available

Reply:
One of the requested changes was: Remove prysm.tmpl from Gnosis envs templates